Image/image.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Prakash Sakari">
+    <title>Responsive Images</title>
+    <link rel="stylesheet" href="image.css">
+</head>
+<body>
+    <div class="container-img-responsive">
+        <img class="img-responsive" src="/assets/shoes.jpg" alt="image">
```

Comment:
Suggestion: Can add a better alt name. It is a given that it is a image. You can describe what image it is. In this case - shoes.